src/commands/job/execute.js

```diff
@@ -47,14 +52,17 @@ export async function execute(
     )
     .catch((err) => {
       result = err
-
+      // need to fix this error return, it's not printing `err` on console
```

Comment:
Should we not just throw an error here? It'll be caught by the layer above and be displayed in the console.

Reply:
updated